src/Utilities/FileSystem.cpp

```diff
@@ -78,35 +78,33 @@ std::string get_absolute_path(const std::string& rel_path) {
           << rel_path << "'.");
       // LCOV_EXCL_STOP
     }
-    ERROR("Failed to get absolute path for an unknown reason.");
+    ERROR("Failed to get absolute path for an unknown reason: " << errno);
```

Comment:
I guess the only way `errno` could get overwritten is if `std::ostringstream` overwrites it when throwing an exception. I have no idea if this is something it is allowed to do or if it's even documented that it's allowed to do that. I'll add an extra variable just in case it can happen.